cypress/integration/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -19,7 +19,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           ({ body }) => {
             const { id, externalId } = body.content.blocks[2];
             cy.get(
-              `iframe[src="${`/ws/av-embeds/media/${externalId}/${id}`}"]`,
+              `iframe[src="${`https://www.test.bbc.com/ws/av-embeds/media/${externalId}/${id}`}"]`,
```

Comment:
Will these need updating once the pages are live?

Could we use something like `iframe[src$=/ws/av-embeds/media/...]` to make it host-agnostic?

Reply:
Potentially something to look into as a follow up issue. Ideally this PR should just stabilise the E2Es

Reply:
Fine with me!